knowledge-base/hide-table-borders-extending-to-the-next-page.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Hide table borders extending to the next page
+description: How to hide table borders that extend to the end of the page
+type: how-to
+page_title: Remove extra table borders when the table is paged
```

Comment:
When it comes to page title, all words should be with capital letters except the prepositions, so it should be Remove Extra Table Borders When the Table is Paged